hydra/experimental/initialize.py

```diff
@@ -29,51 +22,31 @@ def restore_gh_from_backup(_gh_backup: Any) -> Any:
 
 
 class initialize:
-    """
-    Initializes Hydra and add the config_path to the config search path.
-    config_path is relative to the parent of the caller.
-    Hydra detects the caller type automatically at runtime.
-
-    Supported callers:
-    - Python scripts
-    - Python modules
-    - Unit tests
-    - Jupyter notebooks.
-    :param config_path: path relative to the parent of the caller
-    :param job_name: the value for hydra.job.name (By default it is automatically detected based on the caller)
-    :param caller_stack_depth: stack depth of the caller, defaults to 1 (direct caller).
-    """
-
     def __init__(
         self,
         config_path: Optional[str] = None,
         job_name: Optional[str] = None,
         caller_stack_depth: int = 1,
     ) -> None:
-        self._gh_backup = get_gh_backup()
+        from hydra import initialize as real_initialize
 
-        if config_path is not None and os.path.isabs(config_path):
-            raise HydraException("config_path in initialize() must be relative")
-        calling_file, calling_module = detect_calling_file_or_module_from_stack_frame(
-            caller_stack_depth + 1
+        warnings.warn(
+            category=UserWarning,
+            message="hydra.experimental.initialize() is no longer experimental."
+            " Use hydra.initialize()",
```

Comment:
Should we add something like: this will be removed in Hydra 1.2?

Reply:
It sounds a bit weird to say that it will be removed in 1.2.